plugins/cisco_umbrella_destinations/unit_test/mock.py

```diff
@@ -0,0 +1,122 @@
+import json
+import os
+from typing import Callable
+from unittest import mock
+from icon_cisco_umbrella_destinations.connection.schema import Input
+
+import requests
+
+STUB_ORG_ID = "1234567"
+STUB_API_KEY = "j5740ps1cbukyk3t8kib3wa36aq2v3da"
+STUB_DESTINATION_LIST_ID = "12345678"
+STUB_CONNECTION = {
+    Input.API_KEY: {"secretKey": STUB_API_KEY},
+    Input.API_SECRET: {"secretKey": STUB_API_KEY},
+    Input.ORGANIZATION_ID: STUB_ORG_ID,
+}
+STUB_RESPONSE = {
+    "success": {
+        "status": {"code": 200, "text": "OK"},
+        "data": {
+            "id": 15755711,
+            "organizationId": 2372338,
+            "access": "allow",
+            "isGlobal": False,
+            "name": "CreateListTest",
+            "thirdpartyCategoryId": None,
+            "createdAt": "2022-01-28T16:03:36+0000",
+            "modifiedAt": "2022-02-09T11:47:00+0000",
+            "isMspDefault": False,
+            "markedForDeletion": False,
+            "bundleTypeId": 1,
+            "meta": {"destinationCount": 5},
+        },
+    }
+}
+BASE_URL = f"https://management.api.umbrella.com/v1/organizations/{STUB_ORG_ID}/destinationlists"
+
+
+class MockResponse:
+    def __init__(self, filename: str, status_code: int, text: str = "") -> None:
+        self.filename = filename
+        self.status_code = status_code
+        self.text = text
+
+    def json(self):
+        with open(
+            os.path.join(os.path.dirname(os.path.realpath(__file__)), f"responses/{self.filename}.json.resp")
```

Comment:
Are you attempting to get this file's parent directory's absolute path? I think it would be better to add the parent directory to the OS `path` with `sys.path.append(os.path.abspath("../"))` like we do in [other](https://github.com/rapid7/insightconnect-plugins/blob/dd1f2e8024c6bce2f29e9222278a4c31ba042a64/plugins/ibm_qradar/unit_test/test_utils.py#L8) plugins. I don't currently have in mind the end-to-end description of how better to get the path to the response files, let me follow up with you on that.

Reply:
For this, I copied how Igor and Lukasz did theirs for the OpsGenie plugin. It seems to work perfectly and not cause any issues